txpool/tx_pool_test.go

```diff
@@ -95,33 +99,29 @@ func TestNewCloseWithServer(t *testing.T) {
 	// Create a slice of transactions to be added to the pool.
 	txs := make(Tx.Transactions, 0, 15)
 	for i := 0; i < 15; i++ {
-		tx := newTx(pool.repo.ChainTag(), nil, 21000, tx.BlockRef{}, 100, nil, tx.Features(0), genesis.DevAccounts()[i%len(genesis.DevAccounts())])
+		tx := newTx(pool.repo.ChainTag(), nil, 21000, tx.BlockRef{}, 100, nil, tx.Features(0), devAccounts[i%len(devAccounts)])
 		txs = append(txs, tx)
 	}
 
 	// Call the Fill method
 	pool.Fill(txs)
 
-	// Add a delay of 2 seconds
-	time.Sleep(2 * time.Second)
+	time.Sleep(1 * time.Second)
```

Comment:
is this sleep to be left ? as rest is removed

Reply:
Generally, I wanted to remove the sleep function. But removing this one will make the coverage drop-down. But as I added more test cases, removing it in this PR won't make the overall coverage down, wdyt?

Reply:
I think it would make sense to remove it, if It wont affect coverage that much

Reply:
Tried locally, and coverage drops from 90% to 86%, I think I am going to keep it

Reply:
yeah 4% is quite high thout it will be somethimg much smaller :)